shard/src/guild_settings.py

```diff
@@ -55,6 +55,24 @@ def starboard_emoji(self, new_emoji: str):
         self._settings_dict['starboard_emoji'] = new_emoji
         self._update_db()
 
+    @property
+    def pug_emoji(self) -> str:
+        return self._settings_dict['pug_emoji'] if 'pug_emoji' in self._settings_dict else "✅"
+
+    @pug_emoji.setter
+    def pugemoji(self, new_emoji: str):
+        self._settings_dict['pug_emoji'] = new_emoji
+        self._update_db()
+
+    @property
+    def pug_timeout_speed(self) -> int:
+        return self._settings_dict['pug_timeout_speed'] if 'pug_timeout_speed' in self._settings_dict else 5
+
+    @pug_timeout_speed.setter
+    def pug_timeout_speed(self, new_severity: int):
+        self._settings_dict['pug_timeout_speed'] = new_severity
+        self._update_db()
+
```

Comment:
you should know that this doesn't get automatically added to the `settings` command.  I'm fine to merge this without it, since they're reasonable defaults, but just fyi

Reply:
its ok its more for later